packages/jest-transform/src/__tests__/script_transformer.test.js

```diff
@@ -541,6 +535,94 @@ describe('ScriptTransformer', () => {
     expect(writeFileAtomic.sync).toHaveBeenCalledTimes(1);
   });
 
+  it('should write a source map for the instrumented file when transformed', () => {
+    config = {
+      ...config,
```

Comment:
Ew, why does each test mutate `config`... I know you didn't introduce this, I just haven't seen it before

Reply:
Fixing here by copying into a `const`.